tests/fail/concurrency/thread_local_static_dealloc.stderr

```diff
@@ -6,7 +6,12 @@ LL |         let _val = *dangling_ptr.0;
    |
    = help: this indicates a bug in the program: it performed an invalid operation, and caused Undefined Behavior
    = help: see https://doc.rust-lang.org/nightly/reference/behavior-considered-undefined.html for further information
-   = note: BACKTRACE:
+help: ALLOC was allocated here:
+  --> $DIR/thread_local_static_dealloc.rs:LL:CC
+   |
+LL |         let dangling_ptr = std::thread::spawn(|| SendRaw(&TLS as *const u8)).join().unwrap();
```

Comment:
This is a slightly suspicious span, though I guess for thread-locals it is unclear what the span *should* be.

Reply:
Yup. No idea what it should be for allocation _or_ deallocation.